.changeset/mean-queens-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-newrelic': patch
+---
+
+Fixed bug in NewRelicComponent component where table would not sort correctly for numerical values.
```

Comment:
```suggestion
Fix sorting bug in `NewRelicFetchComponent` table by making all values numerical.
```

Reply:
Thank you for the fast review! Look forward to contributing more 🎉